cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -69,6 +86,12 @@ The application launches a MapReduce program that runs every minute, reads data
 stream *myStream* and writes to a Table *myTable*. A Table Sink needs a row key field to
 be specified and can use the timestamp of a Stream event for that.
 
+Pipeline config consists of stages - source, zero (or) more transforms and one (or) more sinks.
+Each of these stages are identified by a unique name and plugin.
+Plugin object consists of plugin-name, properties map and optionally can specify the artifact.
```

Comment:
we also need a lot more detail here about connections.  We need to take about splitting, and the fact that the same record will be sent to all outputs.  Also need to talk about merging, and how the input schemas must be the same, and how order is undefined, and records are not joined, they are just sent.  Then we need an example or two to demonstrate how it all works.